src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/Converter.cs

```diff
@@ -387,13 +393,19 @@ private static void UpdateTestResultAttachments(ObjectModel.TestResult rockStead
                         TrxLoggerResources.FailureToAttach,
                         attachmentSet.DisplayName, 
                         e.GetType().ToString(), 
-                        e.Message);
+                        e);
+
+                    if (ObjectModel.EqtTrace.IsErrorEnabled)
+                    {
+                        ObjectModel.EqtTrace.Error("Converter: UpdateTestResultAttachments: " + errorMsg);
+                    }
 
                     StringBuilder stdErr = new StringBuilder(testResult.StdErr);
                     stdErr.AppendLine(errorMsg);
 
                     testResult.StdErr = stdErr.ToString();
                     testResult.Outcome = TrxObjectModel.TestOutcome.Error;
+                    throw e;
```

Comment:
Add rethrow `throw;`.

Reply:
Removing throw. Keeping the behavior same as before